samza-core/src/main/java/org/apache/samza/config/TaskConfigJava.java

```diff
@@ -117,4 +121,15 @@ public TaskConfigJava(Config config) {
 
     return Collections.unmodifiableSet(allInputSS);
   }
+
+  /**
+   * Returns a value indicating how long to wait for the tasks to shutdown
+   *
+   * @return value indicating how long to wait for the tasks to shutdown
+   */
+  public long getShutdownMs() {
+    String shutdownMs = get(TASK_SHUTDOWN_MS);
+    if (shutdownMs == null)  return DEFAULT_TASK_SHUTDOWN_MS;
+    return Long.parseLong(shutdownMs);
```

Comment:
I didn't get it from your first comment. I was trying to keep the behavior same as before. Fixing it now. Your suggestion is better :)